google-cloud-spanner-hibernate-dialect/src/test/java/com/google/cloud/spanner/hibernate/SpannerForeignKeyExporterTests.java

```diff
@@ -57,29 +58,32 @@ public void setup() {
     when(spannerDatabaseInfo.getImportedForeignKeys("address"))
         .thenReturn(Collections.singleton("address_fk"));
 
-    this.metadata = setupMetadataMock();
+    this.metadata = mock(Metadata.class);
     this.spannerForeignKeyExporter = new SpannerForeignKeyExporter(new SpannerDialect());
     this.spannerForeignKeyExporter.init(spannerDatabaseInfo);
+    this.context = mock(SqlStringGenerationContext.class);
+    when(this.context.format(any(QualifiedTableName.class))).thenAnswer(invocation ->
+        ((QualifiedTableName) invocation.getArguments()[0]).getTableName().getCanonicalName());
   }
 
   @Test
   public void testDropForeignKey() {
     String[] dropStatements = spannerForeignKeyExporter.getSqlDropStrings(
-        foreignKey("address", "address_fk"), metadata);
+        foreignKey("address", "address_fk"), metadata, this.context);
```

Comment:
Fixes test that was resulting `"getSqlDropStrings() was not implemented!"` error message which was caused due to deprecation of `getSqlDropStrings(T exportable, Metadata metadata)`